app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+from app.players.player import Player
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
```

Comment:
here you don't need to import classes

Reply:
I will have an arrors, becuse i have they as arg annotations list[Player], list[Elf] ....

Reply:
from abc import ABC is necessary!!! Code will be work without that. But today with Danya we discussed and he said that it is a good practice to write because Elf is abstract class and you can not remember it in big project. 
But when you see ABC,  you know that it is abstract. I think it is right.

Reply:
> I will have an arrors, because i have they as arg annotations list[Player], list[Elf] ....
Ok. I did not notice these annotations...